doc/release-notes-16512.md

```diff
@@ -0,0 +1,4 @@
+RPC changes
+-----------
+The RPC `joinpsbts` will shuffle the order of the inputs and outputs of the resulting joined psbt.
```

Comment:
16512 is either incomplete or dnm

Reply:
It seems like all the additions to joinpsbts was not added which could explain why including random did nothing